README.md

```diff
@@ -1 +1,55 @@
-Hello
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Awesome Books
+
+> Awesome Books
+
+![screenshot](./app_screenshot.png)
+
+This project allows you to add/remove books 
+
+## Built With
+
+- HTML, CSS, and JavaScript
+
+## Live Demo
+
+[Portfolio Web Page](https://amrhub.github.io/myPortfolio/)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Setup
+
+To setup the webpage in your local up in the repo page:
+click on code (dropdown list) > Download as ZIP
+
+### Deployment
+
+To run the project open folder (after extraction) then run index.html
+
+## Authors
```

Comment:
- [ ] OPTIONAL
Kindly add the details of the **second contributor** to this project.